From 4add586753c50b3222c0899ced0d95a2263828c6 Mon Sep 17 00:00:00 2001
From: Jakub Hrozek <jhrozek@redhat.com>
Date: Tue, 6 Sep 2016 12:27:51 +0200
Subject: [PATCH 55/79] KRB5: Return ERR_NETWORK_IO on clock skew
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Adds two more return codes to the list of codes we translate to
ERR_NETWORK_IO.

Resolves:
https://fedorahosted.org/sssd/ticket/3174

Reviewed-by: Pavel Březina <pbrezina@redhat.com>
Reviewed-by: Sumit Bose <sbose@redhat.com>
(cherry picked from commit d3348f49260998880bb7cd3b2fb72d562b1b7a64)
---
 src/providers/krb5/krb5_child.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/providers/krb5/krb5_child.c b/src/providers/krb5/krb5_child.c
index a0a0f74d7e39866828c1c9ee4b18e57c36a30bb9..82522995e310f20c58922f814e14e81a84b9bcb9 100644
--- a/src/providers/krb5/krb5_child.c
+++ b/src/providers/krb5/krb5_child.c
@@ -1374,6 +1374,8 @@ static errno_t map_krb5_error(krb5_error_code kerr)
 
     case KRB5_KDCREP_SKEW:
     case KRB5KRB_AP_ERR_SKEW:
+    case KRB5KRB_AP_ERR_TKT_EXPIRED:
+    case KRB5KRB_AP_ERR_TKT_NYV:
     case KRB5_KDC_UNREACH:
     case KRB5_REALM_CANT_RESOLVE:
     case KRB5_REALM_UNKNOWN:
-- 
2.9.3

